rust/datafusion/src/execution/context.rs

```diff
@@ -995,16 +1034,16 @@ mod tests {
         let ctx = create_ctx(&tmp_dir, 1)?;
 
         let schema = Arc::new(Schema::new(vec![
-            Field::new("state", DataType::Utf8, false),
-            Field::new("salary", DataType::UInt32, false),
+            Field::new("c1", DataType::Utf8, false),
+            Field::new("c2", DataType::UInt32, false),
         ]));
 
         let plan = LogicalPlanBuilder::scan("default", "test", schema.as_ref(), None)?
             .aggregate(
-                vec![col("state")],
```

Comment:
This change was necessary because the file's schema is `c1`, `c2`, not other schema. We were able to get away with this because, since the names did not mean anything, we could read a file with a given schema using another schema's field names, as long as the order was correct. I believe that this should not be possible by design, as it introduces situations that are non-trivial to debug.